pkg/scheduler/plugins/predicates/proportional.go

```diff
@@ -35,6 +56,10 @@ func checkNodeResourceIsProportional(task *api.TaskInfo, node *api.NodeInfo, pro
 		}
 	}
 
+	if shouldIgnoreProportion(task) {
```

Comment:
Parsing the annotation during predicating each node may sacrifice the performace.
eg: a cluster with 10000 nodes, it will parse at most 10000 times.

Reply:
did you mean a cluster with 10000 pods (not nodes)?

Reply:
And by the implementation it will be parsed only on pods where such annotation exists, in other cases it not parse anything.

Reply:
@lowang-bh see my answers please

Reply:
This is also ok. I means another way is to parse it when added to job and store in task's info. But it will takes a bit more memory.